tests/post_training/data/wwb_ref_answers/tinyllama__tinyllama-1.1b-step-50k-105b/ref_qa_onnx.csv

```diff
@@ -0,0 +1,179 @@
+,prompts,answers,language
```

Comment:
@alexsu52 The `req_qa_onnx.csv` file was obtained using the following "gold" model:
```python
model_gold = ORTModelForCausalLM.from_pretrained(
    self.fp32_model_dir,
    trust_remote_code=True,
    provider="OpenVINOExecutionProvider"
)
```
It is identical to `ref_qa.csv`, which was generated using the model below:

```python
model_gold = OVModelForCausalLM.from_pretrained(
    self.fp32_model_dir,
    trust_remote_code=True,
    load_in_8bit=False,
    compile=False,
    stateful=is_stateful,
    ov_config={"KV_CACHE_PRECISION": "f16"},
)
````
So, if you don't mind, I suggest using only the `ref_qa.csv` file.

Reply:
ok